style.css

```diff
@@ -68,6 +78,227 @@ body {
+  width: 98%;
+  height: auto;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.content-placeholder {
+  margin: auto;
+  width: 90%;
+}
+
+.inline,
+.inline-li li {
+  display: inline;
+  padding-right: 7px;
+}
+
+.social-media ul {
+  display: flex;
+  gap: 20px;
+  margin-top: 0.75em;
+}
+
+.pl-1 {
+  padding-left: 1em;
+}
+
+.project-summary > li::before {
+  content: '\2022';
+  margin-right: 0.5em;
+  color: #7a869a;
+}
+
+li.client::before {
+  content: '';
+  margin-right: 0;
+}
+
+.project-summary { color: #7a869a; }
+
+.client { color: #344563; }
+
+.social-media .inline-li { padding: 0 30px; }
+
+.social-items img {
+  width: 20px;
+  height: 20px;
+}
+
+.tags {
+  height: 25px;
+  border: 1px solid #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  background-color: #ebebff;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+}
+
+.btn {
+  width: fit-content;
+  height: 48px;
+  border-radius: 8px;
+  padding: 12px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: center;
+  margin: 30px 0;
+}
+
+.btn:enabled {
+  border: 1px solid #6070ff;
+  color: #396df2;
+  background-color: #fff;
+}
+
+.btn:hover {
+  background-color: #6070ff;
+  color: #fff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+  transition: all 1s;
+}
+
+.btn:disabled {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
+
+.btn:active { color: #fff; }
+
+#about {
+  background-color: #fff;
+  border-top-right-radius: 100px;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 50px 10px;
+}
+
+.language-ul {
+  width: 99%;
+  padding: 0;
+  list-style: none;
+  display: block;
+}
+
+.language-ul li {
+  margin: 2px auto;
+  padding: 5px;
+  line-height: 25px;
+  background-color: #f7f7f9;
+  text-align: center;
+  align-items: center;
+  gap: 20px;
+  display: flex;
+}
+
+.skills-dropdown {
+  width: 99%;
+  height: 48px;
+}
+
+.skills-dropdown:first-child {
+  height: auto;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+#frameworks,
+#skills { border-bottom: 1px solid #dfe1e6; }
+
+.flex {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 20px;
+  padding: 0.7% 3.4%;
+}
+
+.skills-dropdown:first-child > div {
+  justify-content: space-between;
+  width: 99%;
+}
+
+.footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #6070ff;
+  border-top-left-radius: 100px;
+  width: 100%;
+  background-image: url(images/contact-bg_01.png), url(images/contact-bg_03.png);
+  background-position: center top, right top;
+  background-repeat: no-repeat;
+}
+
+footer .content-placeholder {
+  text-align: center;
+  padding: 60px 5px;
+}
+
+#contact-msg,
+#contact-me { color: #fff; }
+
+form {
+  width: 100%;
+  background-image: url(images/contact-bg_02.png);
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  flex-wrap: wrap;
+}
```

Comment:
Please consider making the styles in accordance with the template to make it look alike the template.

## Solution:---

```
font-family: 'Poppins';
```